### DISCUSSION TITLE
POI file France / Germany too big

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#159] [Q&A] [answered]
The POI file for France is suddenly 13 GB. This is too big for download.

**Comment by MotoUKRider** [comment]
That's a lot of cafes!

**Reply by 0709wiwiwi** [reply]
Abundance especially considering Brits only need one  ;-)

**Comment by devemux86** [comment, marked as answer]
Thanks, this is fixed, the .poi files will be updated.

Related to the new POI v3 format:
- https://github.com/devemux86/cruiser/discussions/151

Cruiser can use multiple POI in offline search (so can load many regions):
- https://github.com/devemux86/cruiser/discussions/155

**Comment by devemux86** [comment]
POI of Germany seems to be finished: 11 GB! :slightly_smiling_face: 
When the size of all individual regions is less than 2GB.

More tests are needed....

Until then please download the POI of the regions.
Anyway now you can activate many POI together.

**Reply by DolomitesTom** [reply]
OK 🙈, thanks for information.

**Comment by devemux86** [comment]
The issue has been fixed and all .poi files are currently being rebuilt.

You will receive updates on the downloaded POI over the next days.

**Comment by devemux86** [comment]
- France 473 MB
- Germany 702 MB

**Reply by DolomitesTom** [reply]
Thai fits

**Comment by DolomitesTom** [comment]
When I use the search function for fuel stations, in the old search routine all those were shown with blue dots. This is something that I am missing, because one can see the search results only in a list and you don't see in which direction from you actual position these are. Can there be something like "show the results on the map"?

**Reply by devemux86** [reply]
@DolomitesTom please see:
- https://github.com/devemux86/cruiser/discussions/74#discussioncomment-5895826
